tests/Unit/HelpersTest.php

```
@@ -0,0 +1,227 @@
+		$this->assertSame( 'v2', $defaults['swh_recaptcha_type'] );
+		$this->assertSame( '', $defaults['swh_recaptcha_project_id'] );
+		$this->assertSame( '', $defaults['swh_recaptcha_api_key'] );
+		$this->assertSame( '0.5', $defaults['swh_recaptcha_threshold'] );
+	}
+
+	/** Result is cached (same instance returned on second call). */
+	public function test_get_defaults_returns_same_instance(): void {
+		$first  = swh_get_defaults();
+		$second = swh_get_defaults();
+		$this->assertSame( $first, $second );
+	}
```

PHPMD (2.15.0)

[error] The method test_get_defaults_returns_same_instance is not named in camelCase. (undefined)

(CamelCaseMethodName)